integration_tests/ScaleTest.md

```diff
@@ -0,0 +1,78 @@
+# Scale Test
+
+## Build
+
+The Scale Test suite is bundled into `integration_tests` module, it can be built by maven directly.
+
+```bash
+mvn package
+```
+
+You can choose profiles to build against specific Spark and JDK version. e.g.
+
+```bash
+mvn package -Dbuildver=332 -Pjdk17
+```
+
+## Query Sets
+
+The queries are define in the source code. You can check the table below to see query content and the description:
+
+| name | description | content |
+|------|-------------|---------|
+| q1   | Inner join with lots of ride along columns | SELECT a_facts.*, b_data_{1-10} FROM b_data JOIN a_facts WHERE primary_a = b_foreign_a |
```

Comment:
I am a little concerned about maintaining queries in multiple places. Do we want to keep all of this here? Including the content column?

Reply:
I put the queries in the doc just to give new viewers the most direct way to know what are the queries. I could also put a link here points to the queries defined in the code which actually help me reduce the copy work :)

Also btw, I'm about to add the timeout field for each query(https://github.com/NVIDIA/spark-rapids/issues/8814#issuecomment-1652368305).

Reply:
We could go crazy with it and auto-generate things, but it might be simpler just to point to the code and add a comment with the code what each query is intended to do.  I think this is fine for now.